### PR TITLE
Employees endpoint

### PR DESCRIPTION
**Pull request by allaire**
Added employee resource, nothing fancy, but we needed it for employees import.

Also added Serializer support, which works well (https://github.com/rails-api/active_model_serializers/issues/568 / https://github.com/rails/rails/blob/4-2-stable/activemodel/lib/active_model/serialization.rb#L141)

### CODE REVIEW

File: lib/lightspeed/resource.rb [hunk 1]
```diff
@@ -12,6 +12,9 @@ class Link; end
   class Resource
     attr_accessor :id, :attributes, :client, :context, :account
 
+    # Serialization support
+    alias :read_attribute_for_serialization :send
```
Comment (dnl) [inline lib/lightspeed/resource.rb]: Hi Allaire 

I'm not entirely comfortable with the all-encompassing nature of this. 

Are you able to hang that off the `as_json` method somehow, possibly even something like

```
def read_attribute_for_serialization(method_name)
end
```

**Comment by dnl** [comment]
I'm happy to add the employee endpoint. I just have that concern about the blanket serialization.

Thanks for the contribution.

File: lib/lightspeed/resource.rb (hunk 1 above)
Comment (allaire) [inline lib/lightspeed/resource.rb]: Sure. I'll implement `read_attribute_for_serialization` tomorrow like you show!

Comment (dnl) [inline lib/lightspeed/resource.rb]: note, I haven't actually run this code, it's a suggestion... etc etc etc.

**Comment by allaire** [comment]
@dnl Hey so that method you pasted worked out perfectly, just tried it out with our API :+1: 

I also pushed my vcr cassettes, is this best practice?

Finally, I used hashes for:

```
      Contact: :hash,
      EmployeeRole: :hash,
      EmployeeRights: :hash
```

But they act a bit to a relationship, in the sense of it's an array when there's multiple items and a hash when there's only one item (not a fan of this API style but oh well). From what I understand relationships are used when the relation is question is another resource in their API. Contact, EmployeeRole and EmployeeRights aren't resources. Should I keep everything like that?

Thanks!

**Comment by dnl** [comment]
The array vs hash won't be an issue, as the json request converts these objects properly before they even get to our conversion. (it's presence in https://github.com/marketplacer/lightspeed_pos/blob/master/lib/lightspeed/resource.rb#L45 is misleading, and it's wrong anyway. :P I'll get around to fixing that at some point)

If you want to have a nicer/more consistent API for these then you might be able to do that similarly to how I dealt with Item's `Price` attribute, but what you have is fine.

If you're happy with the information that is presently visible in the VCRs being in the codebase (there are phone numbers and names and emails, not sure if test data or real data), then let me know and I'll merge this as is, otherwise please modify the VCR yaml files directly to anonymize them.

**Comment by allaire** [comment]
Good point! Should be all good to merge @dnl 

Thanks!

**Pull request merged**